### PR TITLE
Tweek review status pages

### PR DESCRIPTION
**Pull request by chrisndodge**

### CODE REVIEW

**Comment by chrisndodge** [comment]
@catong I just realized you weren't in the loop on a new feature which will allow students to review his/her proctored exam after the proctored review has been received. This corresponds to PHX-216.

Since this PR is some content tweeks I'm doing, you might want to look at the full rejected.html, submitted.html, verified.html, and visit_exam_content.html

**Comment by chrisndodge** [comment]
@afzaledx @muhhshoaib while we wait for Documentation's feedback, can you review?

File: edx_proctoring/templates/proctored_exam/visit_exam_content.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {% load i18n %}
 {% if has_due_date_passed %}
-
+  <hr>
   <p>
     {% blocktrans %}
-      If you want to view your exam questions and responses, click on the button “let me see my exam”.
-      When you clicked the button, you will be taken directly to the exam content. The exam’s status will
-      still be visible to you in the left navigation pane.
+      If you want to view your exam questions and responses, click on the “Let me see my exam” button below.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: "To view your exam questions and responses, select **Let me see my exam**." The exam's review status is shown in the left navigation pane."
@chrisndodge for line 6-8. Please use boldface for the button text (indicated with double asterisks in this comment).

File: edx_proctoring/templates/proctored_exam/visit_exam_content.html [hunk 2]
```diff
@@ -37,4 +37,10 @@
       }
     );
   </script>
+{% elif has_due_date %}
+  <p>
+    {% blocktrans %}
+      Once the due date for this exam has passed, you will be able to go visit this page and review your answers.
```
Comment (catong) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: "After the due date for this exam has passed, you will be able to review your answers on this page."

File: edx_proctoring/templates/proctored_exam/visit_exam_content.html (hunk 1 above)
Comment (chrisndodge) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: Is "Let me see my exam" supposed to render as bold in the page?

Comment (catong) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: @chrisndodge I also have a suggestion for the "Let me see my exam" button text -> "View my exam".
If you go with this suggestion, then also modify the "To view your exam questions..." text from my comment above.

**Comment by catong** [comment]
@chrisndodge Left my edits in comments

File: edx_proctoring/templates/proctored_exam/visit_exam_content.html (hunk 1 above)
Comment (catong) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: @chrisndodge It should be consistent with other places where we quote button text -- either bf or double quotation marks?

File: edx_proctoring/templates/proctored_exam/visit_exam_content.html (hunk 2 above)
Comment (chrisndodge) [inline edx_proctoring/templates/proctored_exam/visit_exam_content.html]: updated

**Comment by chrisndodge** [comment]
@catong are you signing off from a Docs point of view or does @pdesjardins need to look as well?

**Comment by catong** [comment]
@chrisndodge I took on this one because Peter was in training yesterday. We're all set for this PR; for other/future items, you can tag @pdesjardins only.

**Pull request merged**